src/functions/functions-api.ts

```diff
@@ -70,3 +106,15 @@ export interface TaskOptionsExperimental {
  */
   uri?: string;
 }
+
+export class TaskAlreadyExists extends FirebaseError {
```

Comment:
I would suggest we reuse the `FirebaseFunctionsError` type for this and add a new error code for `task-already-exists` error response to keep it consistent with the rest of the services.